randhound/randhound.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/dedis/onet"
 	"github.com/dedis/onet/log"
 	"github.com/dedis/onet/network"
+	"gopkg.in/dedis/crypto.v0/abstract"
```

Comment:
that's probably wrong here

Reply:
This is dead code that is waiting to be revived. I thought you were working on that?